### PR TITLE
Update to 1.2.6

### LINKED ISSUE #51: problem packages for e3sm-unified 1.2.6
A number of packages are giving me trouble for e3sm-unified 1.2.6:
- [x] `output_viewer` - no `conda-forge` build for python 3.7.  Need this PR merged: https://github.com/conda-forge/output_viewer-feedstock/pull/4.  This is also a dependency of `e3sm_diags`, which is a dependency of `processflow`, so neither of those can be included without this fix.
- [x] `thermo` - conflicting package in `conda-forge` channel is taking priority.  This package probably needs to be renamed.  Issue here: https://github.com/CDAT/thermo/issues/11.  Dependency of `cdat`, so that also currently cannot be included. Moved to `vcsaddons` by @doutriaux1, thanks!
- [x] `globus-cli` - dependency `jmespath=0.9.2` not build for python 3.7.  Pull request in here: https://github.com/conda-forge/jmespath-feedstock/pull/7. Issue requesting an update in dependency here: https://github.com/globus/globus-cli/issues/461
- [x] `cdp` - built incorrectly with pinned python.  Fixed via: https://github.com/conda-forge/cdp-feedstock/pull/7
- [x] `processflow` - depends on `globus-cli 1.1.2` and `globus-sdk 1.1.1`, neither of which is available on conda-forge.  With strict channel priority (needed to make builds work with the latest conda), these versions from the `e3sm` channel are no longer available to `e3sm-unified`. Here's the build of `globus-sdk 1.1.1`: https://github.com/conda-forge/globus-sdk-feedstock/pull/8 and of `globus-cli 1.1.2`: https://github.com/conda-forge/globus-cli-feedstock/pull/6
- [x] `ncl` vs. `basemap` - `ncl` is a dependency of `processflow`; `basemap` is a dependency of `ilamb`, `mpas-analysis` and `cdat`.  `ncl` is not compatible with `pyproj>=1.9.6,<2.0.0` and `proj4>=5.2.0,<6.0.0` for any build with `hdf5=1.10.5`.  `basemap` has not yet been released with a version compatible with `pyproj>=2.0.0` and `proj4>=6.0.0`.  A request was made over a year ago to have a new release of `basemap` with no luck: https://github.com/matplotlib/basemap/issues/405.  This issue was addressed with a new release of `processflow` without the `ncl` dependency.  Users will need to load the local system version manually.
- [x] `hdf5=1.10.5` - Not a show-stopper but a pain is the dependence of various packages on the `cdat/label/v81` channel on `hdf5=1.10.4`, whereas conda-forge packages are being built with `hdf5=1.10.5`.  I had to rebuild `mpas_tools=0.0.3` and `nco=4.8.1` with this older version.  A request to have `cdat/label/v81` packages rebuilt with the more recent hdf5 is here: https://github.com/CDAT/cdat/issues/2234

**Update**: I tried using `hdf5=1.10.5` this morning and it "just worked"!  But there's a remaining incompatibility between `basemap` and `ncl` that I don't have a good solution for.

### PR DESCRIPTION
**Pull request by xylar**
Update packages for v1.2.6

### Package changes
main packages:
* `e3sm_diags` - 1.6.1 (py2), 1.7.1 (py3)
* `geometric_features` - 0.1.1
* `mpas-analysis` - 1.2.4
* `mpas_tools` - 0.0.3
* `nco` - 4.8.1
* `zstash` - 0.2.0 (py2), 0.3.0 (py3)
* `processflow` - 2.2.1 (py2)

dependencies:
* `output_viewer` - 1.3.0
* `xarray` - 0.11.3 (py2), 0.12.3 (py3) 
* `dask` - 2.1.0
* `pyproj` - >=1.9.6,<2.0.0
* `proj4` - >=5.2.0,<6.0.0
* `hdf5` - 1.10.5
* `netcdf4` - 1.5.1.2
* `jigsaw` - nospec
* `metis` - nospec
* `pyflann` - nospec 
* `scikit-image` - nospec
* `pyamg` - nospec
* `ffmpeg` - nospec

`processflow` has been moved to its own separate environment because of incompatibilities between `ncl` and `basemap`, see #51 

closes #51

### CODE REVIEW

**Comment by xylar** [comment]
@rpwagner, looks like CI is failing because we've lost our access token.  Care to take a look?  This version of e3sm-unified is a long way from ready, so no rush.

**Comment by xylar** [comment]
Not ready to merge until #51 gets addressed.

**Comment by rpwagner** [comment]
It's failing due to a chicken and egg problem:
```
Solving environment: ...working... failed
PackagesNotFoundError: The following packages are not available from current channels:
  - e3sm-unified=1.2.6
```

I was thinking about this, and the definition file for CI shouldn't have a pinned version. That doesn't make sense. We can pin versions for releases.

File: e3sm-unified.def [hunk 1]
```diff
@@ -11,7 +11,7 @@ From: continuumio/miniconda
 
 %post -c /bin/bash
     . /opt/conda/etc/profile.d/conda.sh
-    conda create -n e3sm-unified-nox -c conda-forge -c e3sm -c cdat/label/v81 python=2.7 e3sm-unified=1.2.5 mesalib
+    conda create -n e3sm-unified-nox -c conda-forge -c e3sm -c cdat/label/v81 python=2.7 e3sm-unified=1.2.6 mesalib
```
Comment (rpwagner) [inline e3sm-unified.def]: @xylar try unpinning the e3sm-unified version.

**Comment by xylar** [comment]
@rpwagner, I see.  I should have realized that the current source code isn't being used to build the package that goes into the module.  I think unpinning the version is probably not a good idea, since it would likely give the misimpression that the current recipe for the conda package is building in CI, which is not the case.  Instead, I think the right solution is to leave the version pinned at 1.2.5 for now and update it once the packages have been uploaded to the e3sm channel.

Thanks for the help!

**Comment by xylar** [comment]
@rpwagner, just FYI, the new container seems to have built successfully now that I uploaded the packages to the e3sm channel.

**Pull request merged**